### PR TITLE
Feature/reveal sections scrolling

### PR DESCRIPTION
**Pull request by cherylli**
|                                    Web Dev Path                                    |
| :--------------------------------------------------------------------------------: |
| [**100**](https://github.com/Web-Dev-Path/web-dev-path/issues/100) |

#### Have you updated the CHANGELOG.md file? If not, please do it.
yes

#### What is this change?
 all the sections but the Hero.js being revealed only on scrolling and with a translateY effect that simulates a smooth movement towards the top

#### Were there any complications while making this change?
no

#### How did you verify this change?
run locally on firefox, chrome, brave and in responsive design mode

#### When should this be merged?
after reviews

### CODE REVIEW

**Comment by cherylli** [comment]
I have added interestion observer (with a custom hook) on TwoColumns and CardColumns. Which means everything with TwoColumns and CardColumns will have this revealing effect, which works fine for now. I'm not sure if this will affect future contents if these components are to be used elsewhere. 

Another way I can think of is create another component with intersect to wrap around individual front page components

File: components/Footer.js [hunk 1]
```diff
@@ -4,39 +4,42 @@ import { linksNav } from '../utils/links';
 import NewsletterSubscribe from './mailchimp/NewsletterSubscribe';
 import Image from 'next/image';
 import Container from './Container';
+import RevealContentContainer from './RevealContentContainer';
```
Comment (mariana-caldas) [inline components/Footer.js:7]: Could you please remove the effect from the footer? Thanks!

**Review by mariana-caldas — CHANGES_REQUESTED** [review]
I just have that single request on the footer, @cherylli !
This looks great, thanks!

File: components/Footer.js (hunk 1 above)
Comment (cherylli) [reply components/Footer.js]: removed

**Review by mariana-caldas — APPROVED** [review]
LG(reat)TM, @cherylli ! 🥳 
Can we just wait a few days to see if someone else has more feedback before you merge it, pls? Thanks!

**Pull request merged**